docs/en/user-guide.md

```diff
@@ -44,3 +45,38 @@ To see more detailed usage, run the flag `-h` or `--help`.
 ## Download
 
 Download the nightly version of Dumpling [here](https://download.pingcap.org/dumpling-nightly-linux-amd64.tar.gz).
+
+## Output filename template
+
+The `--output-filename-template` argument specifies how all files are named, before including the file extensions. It accepts strings in the [Go `text/template` syntax](https://golang.org/pkg/text/template/).
+
+The following fields are available to the template:
+
+* `.DB` — database name
+* `.Table` — table name or object name
+* `.Index` — when a table is split into multiple files, this is the 0-based sequence number indicating which part we are dumping
+
+The database and table names may contain special characters like `/` not acceptable in the file system. Thus, Dumpling also provided a function `fn` to percent-escape these special characters:
+
+* U+0000 to U+001F (control characters)
+* `/`, `\`, `<`, `>`, `:`, `"`, `*`, `?` (invalid Windows path characters)
```

Comment:
In some japan system ASCII `\` is replaced by `¥`, not sure our hardcode will work

https://en.wikipedia.org/wiki/Yen_sign

Reply:
@lance6716 in Japan and Korea the character `\` is *displayed* as ¥ or ₩, but the character code is still 0x5C